### DISCUSSION TITLE
How can I change spa client to show F instead of C

### DISCUSSION THREAD

**Discussion by michaelmaulucci** [plmilord/Hass.io-custom-component-spaclient#56] [Q&A]
I have my HA setup for Celsius as that is what I normally use for all climate readings, but for my hot tub, I actually like to use Fahrenheit instead.  I can’t seem to find a place to switch it for this entity.  Can someone direct me how to change it.

**Comment by plmilord** [comment]
I'm in the same boat as you... The default for us here is Celsius, but for spa, we use Fahrenheit. I resigned myself to keeping my HA in Fahrenheit.

I've considered that there's probably a possibility to force unity for each integration, but I've never had any success. Maybe there's something missing in my code...

I'll keep this need for future improvement! In the meantime, if you want to explore and share, it would be appreciated.
